tests/layer_tests/py_frontend_tests/test_torch_frontend.py

```diff
@@ -754,3 +758,47 @@ def test_inlined_inputs():
     model.eval()
     model = torch.compile(model, backend="openvino", options={"testing": 1})
     model()
+
+
+class ComplexModel(torch.nn.Module):
+    def __init__(self, dtype):
+        super().__init__()
+        self.dtype = dtype
+        x = torch.tensor([1.0 + 2.0j, 3.0 + 4.0j], dtype=dtype)
+        self.complex_attr = torch.nn.Parameter(x)
+
+    def forward(self, x):
+        complex_const = torch.tensor(
+            [5.0 + 6.0j, 7.0 + 8.0j], dtype=self.dtype)
+
+        real_attr = self.complex_attr.real
+        imag_attr = self.complex_attr.imag
+
+        real_const = complex_const.real
+        imag_const = complex_const.imag
+
+        real_result = x + real_attr + real_const
+        imag_result = imag_attr + imag_const
+
+        result = real_result + imag_result
+        return result
+
+
+@pytest.mark.parametrize("dtype", [
+    torch.complex32,
+    torch.complex64,
+    pytest.param(torch.complex128,
+                 marks=pytest.mark.xfail(reason="CVS-163022"))])
```

Comment:
after merge https://github.com/openvinotoolkit/openvino/pull/29212 xfail should be removed

Reply:
Right